docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,252 @@
+# Natural Language Processing (NLP)
+
+
+NLP is a subset of Natural Language Toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (NLP) functionalities, such as language detection, parts of speech, word tokenize, lemmatize, and named entity detect   ion. For more information, see the [NLTK Forum](http://www.nltk.org/).
+
+The main features of the Tizen.Nlp namespace include:
+
+-   Word Tokenize suppport
+
+    You can get tokens from a sentence, the type of return is [WordTokenizeResult](#wordtokenize). 
+    This method breaks up the sentence into words and punctuation.
+
+-   Part of Speech support
+
+    You can get tokens and tags from a sentence, the type of return is [PosTagResult](#postag). This method breaks up the sentence into words and punctuation with tags attributes.
+
+-   Named Entity Recognizer support
+
+    You can get tokens and tags from a sentence, the type of return is [NamedEntityRecognitionResult](#nechunk). This method breaks up the sentence into words and punctuation with tags attributes.
+
+-   Lemmatizing support
+
+    You can get the actual word from a word, the type of return is [LemmatizeResult](#lemmatize). This method is used to get the actual word from a word.
+
+-   Language Detect support
+
+    You can get language from a sentence, the type of return is [LanguageDetectedResult](#langdetect). This method is used to detect the language of a sentence.
+
+
+## Prerequisites
+
+To enable your application to use the NLP functionality:
+
+1.  Using the [Tizen.Nlp](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Nlp.html) namespace, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
To use

Reply:
@zg2nets,
as this fix was requested on line 34, but applied to line 32,
would you please revert line 32, (use -> enable )
and fix "Using" on line 34 to "To use"?